src/pages/case-studies.js

```diff
@@ -2,118 +2,131 @@ import React, { useState } from 'react';
+            url
+            alternativeText
+            caption
+          }
+          Client_Name
+          Path
+          Service_Category {
+            Category
+          }
+          Sort_Order
         }
-    };
-    let cases;
-    if (filterState) {
-        caseStudies.filter((caseStudy) => caseStudy.Service_Category.map(category => category.Category === (filterState)));
-    } else {
-        cases = caseStudies;
+      }
     }
-
-    return (
-
-        <RedLayout>
-
-            <Hero
-                title='Work that makes a difference'
-                description='Our work impacts the daily lives of millions of people. See how we’ve helped agencies build resilient services at scale.'
-            />
-            <Helmet>
-                <title data-react-helmet="true">Case Study Landing page</title>
-            </Helmet>
-
-            <ul>
-                {categories.map((item, index) => (
-                    <li>
-                        <input
-                            type="checkbox"
-                            id={categories[index].id}
-                            name={categories[index].name}
-                            value={categories[index].value}
-                            onChange={(e) => {
-                                callback(e);
-                            }}
-                        />{categories[index].value}
-                    </li>
-                ))}
-            </ul>
-            <CaseStudyTeasers
-                cases={cases}
-                />
-
-            <PrimaryPageCTA
-                title='Let’s build a public success story.'
-                subtitle='Get in touch to start.'
-                primaryButtonText='Put us to work'
-                secondaryButtonText='Join our team'
+  `);
+
+  const caseStudies = data.allStrapiCaseStudy.nodes;
+
+  const categories = [
+    { id: 1, value: 'Web & CMS', name: 'Web_CMS', isChecked: false },
+    {
+      id: 2,
+      value: 'IT & Service Modernization',
+      name: 'IT_Service_Modernization',
+      isChecked: false,
+    },
+    {
+      id: 3,
+      value: 'Product & Design',
+      name: 'Product_Design',
+      isChecked: false,
+    },
+    {
+      id: 4,
+      value: 'Security & Compliance',
+      name: 'Security_Compliance',
+      isChecked: false,
+    },
+    { id: 5, value: 'Data Services', name: 'Data_Services', isChecked: false },
+    {
+      id: 6,
+      value: 'Workforce Development',
+      name: 'Workforce_Development',
+      isChecked: false,
+    },
+  ];
+
+  const [filterState, setFilterState] = useState(null);
+
+  const callback = (e) => {
+    // console.log(e);
+
+    if (e.target.name !== 'undefined') {
+      if (filterState === e.target.name) {
+        // The button was already selected.
+        setFilterState(null);
+      } else {
+        setFilterState(e.target.name);
+      }
+    }
+  };
+  let cases;
+  if (filterState) {
+    cases = caseStudies.filter((caseStudy) => {
```

Comment:
@jaube05 the cases error seemed to relate to cases not having been defined in line 78.